paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -48,8 +48,8 @@ def add_subparser(subparsers):
         required=True,
     )
     list_parser.add_argument(
-        '-l', '--clusterinstance',
-        help='Mark the service ready for deployment in this clusterinstance (e.g. '
+        '-l', '--deploy_group', '--clusterinstance',
+        help='Mark the service ready for deployment in this deploy group (e.g. '
              'cluster1.canary, cluster2.main)',
```

Comment:
Make a note here that `--clusterinstance` is deprecated in favor of `--deploy_group`; I'm in favor of backwards compatibility here, but we should clean up the old option name at some point.